data/json/monsters/zed_misc.json

```diff
@@ -242,7 +242,7 @@
     "id": "mon_zombie_brute_grappler",
     "type": "MONSTER",
     "name": { "str": "zombie wrestler" },
-    "description": "A slab of festering muscle the size of a well-toned bodybuilder.  Seems eager to strangle the life from you.",
+    "description": "A slab of festering muscle the size of a well-toned bodybuilder.  Its bulky arms are compressed like some sort of accordion.",
```

Comment:
I am unsure about this one.

Reply:
yeah i think im going to scrap that one, i really feel like that zed needs a better description however